src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/external/UserService.kt

```diff
@@ -30,7 +53,75 @@ class UserService(
     )
   }
   fun disableUserByUserId(userId: UUID, deactivateReason: DeactivateReason) =
-    userApiService.disableUserById(userId, deactivateReason)
+    externalUsersApiService.disableUserById(userId, deactivateReason)
+
+  fun amendUserEmailByUserId(
+    userId: UUID,
+    emailAddressInput: String?,
+    url: String,
+  ): String {
+
+    val user = externalUsersSearchApiService.findByUserId(userId)
```

Comment:
Should this now be using the external users service client to determine the userId (and the external users endpoint having a role on it?)

Reply:
Not sure, but could we pick this up under a separate ticket if it is going to need changes to external-users-api too?

Reply:
Discussed with @karenmillermoj - both end points have same roles therefore we agreed OK